### PR TITLE
Languages updates for the draft text

### PR DESCRIPTION
**Pull request by martijnrusschen**
Made some minor edits to improve the language.

### CODE REVIEW

File: draft-foudil-securitytxt.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ the preferred steps for researchers to take when trying to reach out.
 As per section 4 of {{!RFC2142}}, there is an existing convention
 of using the <SECURITY@domain> email address for communications regarding
 security issues. That convention provides only a single, email-based
-channel of communication for security issues per domain, and does not provide
+a channel of communication for security issues per domain, and does not provide
```
Comment (nightwatchcyber) [inline draft-foudil-securitytxt.md]: I think this "a" is extra, there is one on the line above

**Comment by EdOverflow** [comment]
Sorry @nightwatchcyber, I forgot to notify you that I instructed @martijnrusschen not to install all the dependencies and that I would generate the text and HTML files for them.

**Comment by nightwatchcyber** [comment]
@EdOverflow and @martijnrusschen - my apoligizes, I wasn't aware of that. I am going to add a new issue to the project to enable automatic regeneration of the TXT and HTML versions via a CI system like I have seen other projects do.

**Comment by martijnrusschen** [comment]
Thanks, good catch!

**Comment by nightwatchcyber** [comment]
@EdOverflow @martijnrusschen Looks good

File: draft-foudil-securitytxt.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ that disclosed security vulnerabilities and worked with you to remediate the iss
 Acknowledgement: https://example.com/hall-of-fame.html
 ~~~~~~~~~~
 
-Example security acknowledgements page:
+Example security acknowledgments page:
```
Comment (EdOverflow) [inline draft-foudil-securitytxt.md]: This does not adhere to the directive (`Acknowledgement:`).

File: draft-foudil-securitytxt.md [hunk 3]
```diff
@@ -270,7 +270,7 @@ eol                    = *WSP \[CR\] LF
 
 field                  = contact-field /
                          encryption-field /
-                         acknowledgement-field /
+                         Acknowledgments-field /
```
Comment (EdOverflow) [inline draft-foudil-securitytxt.md]: `s/Acknowledgments-field/acknowledgments-field`

**Pull request merged**